### PR TITLE
Make the switch to implementing own stubs easier

### PR DESCRIPTION
**Pull request by wolf99**
This PR is opened with regard to #252 where it was decided to add headers for the core exercise `isogram` and the non-core exercises that it unlocks, and also to add `hints.md` files for the following core exercise `gigasecond` and the non-core exercises that _it_ unlocks to explain how a student should add their own headers.

I have so far added the stub headers to `isogram`, `acronym` and `pangram`.
I have also added a first draft of a `hints.md` file to `gigasecond`

The `hints.md` file also needs to be added to `space-age` and `meetup`. I delay in adding them to make any edits that anyone suggests to the `gigasecond` one before copying it over.

Also, I am not sure how to correctly regenerate the README.md. I have downloaded the `configlet` and unzipped it, then cloned the `problem-specifications` _repository_ to the same directory (which also holds the `c` repository). Then in the root of the `c` repository I run 

```bash
../configlet generate . --only gigasecond --spec-path ../problem-specifications
```

(all as outlined here: https://github.com/exercism/java/issues/1085)
But nothing seems to happen and the `README.md` remains unchanged... What have I done wrong?

### CODE REVIEW

**Comment by wolf99** [comment]
Figured out the `configlet` thing. The `hints.md` needs to be in a `.meta` sub-directory within the exercise directory.

Would like feedback on the `hints.md` itself still though 😃

**Comment by ryanplusplus** [comment]
I made a quick commit that addresses a few typos and fixes up a bit of formatting. Please have a look and make sure I didn't screw anything up. Otherwise this looks good to me.

File: exercises/gigasecond/.meta/hints.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+To check that you have correctly determined the required declaration, just run the tests again and analyse any errors similarly.
+
+Additionally, remember to add any includes to the header file any headers that it itself requires.
+
+
+### Include guards
+
+Before you are finished with the header file, you should add include guards.
+Include guards are used to help prevent the file from being included multiple times by accident.
+If the file _was_ included multiple times then the functions and other items would be declared twice; In C this is an error because the compiler thinks you are trying to declare another _different_ function that has the same name.
```
Comment (ryanplusplus) [inline exercises/gigasecond/.meta/hints.md]: The `;` could probably just be a `.`, but I tried to take only a half step away from the em dash.

**Comment by wolf99** [comment]
Thanks @ryanplusplus I'll add hints to the other two relevant exercises and regenerate the README.md files and remove "WIP" on this PR.

**Comment by ryanplusplus** [comment]
LGTM

**Pull request merged**